ui/package.json

```diff
@@ -25,7 +25,8 @@
     "test": "concurrently \"npm run build:watch\" \"karma start karma.conf.js\"",
     "pretest:once": "npm run build",
     "test:once": "karma start karma.conf.js --single-run",
-    "lint": "tslint ./src/**/*.ts -t verbose",
+    "lint": "yarn ng lint",
+    "lint:fix": "yarn ng lint --fix",
```

Comment:
I'm with @gashcrumb on this: here's no point on mixing up the _npm_ and _yarn_ CLIs in the same project.

Since apparently all the other commands rely on `npm`, I'd stick to it for the time being. You can replace the LOCs above by something around the lines of:

```
"lint": "tslint --project src/tsconfig.json",
"lint:fix": "tslint --project src/tsconfig.json --fix",
```
So you can run `$ npm run lint` to trigger automated code linting.

BTW, there should only be ONE `tsconfig.json` file per project but I've found two. Pick the one that contains your desired setup and remove the other one, moving the final config file to the UI project root folder.

Reply:
Or you can also leave those two command lines as is and then replace all the others that point to `npm run <command>` scripts. Perhaps this will be more convenient in the long run ;)

Reply:
Ugh didn't notice there's `src/tsconfig.json`, I'll check and migrate to `/tsconfig.json`.
For the npm/yarn thing, we also have a plan to migrate to yarn completely - #80
But if that mix up may cause a problem even for a short term, I'll change it to use npm for now. WDYT?

Reply:
`npm` and `yarn` feature pretty much the same API, except when it comes to install, uninstall and run binaries. I'd go ahead with it ;)